woocommerce/pinterest-for-woocommerce
# Updates the readme.txt to meet WordPress requirements

### Changes proposed in this Pull Request:



Closes #921.

This PR updates the `readme.txt` file to meet the WordPress requirements.

* Keeps only the latest changelog in the description section and moves the rest to `changelog.txt`.
* Updates the short description by removing the `entire` word.
* Updates the tags. Ref: p1709647885295299/1709213275.732699-slack-CK365S85V

### Detailed test instructions:


1. Checkout the readme.txt file from this PR
2. Ensure the WordPress requirements are met as described in #921.


### Changelog entry

> Doc - Updates the readme.txt file to meet WordPress requirements
